### ISSUE TITLE
Release 0.6.7 has caused all our pipelines to fail due to the "az login"

### ISSUE THREAD

**Issue by scott-doyland-burrows** [charleszipp/azure-pipelines-tasks-terraform#196]
Hi,

The change below has caused all our pipelines to fail as we do not have "az" installed on our build servers.

https://github.com/charleszipp/azure-pipelines-tasks-terraform/pull/193

I can understand the requirement for some people needing to run "az login", but could this be a "tick box" option please.

**Comment by charleszipp** [comment]
Will add something to make this optional. Thanks for reporting the issue

**Comment by charleszipp** [comment]
@scott-doyland-burrows posting sample from the PR i just opened. Let me know if you believe this should address your issue. This should allow your pipelines, as configured prior to 0.6.7, to run successfully without any changes to the pipeline itself.

runAzLogin input has been added to indicate when az login should be run

Setting to true will indicate az login should be run
```yaml
    - task: TerraformCLI@0
      displayName: 'terraform apply'
      inputs:
        command: apply
        workingDirectory: $(test_templates_dir)
        environmentServiceName: 'env_test'
        commandOptions: '$(System.DefaultWorkingDirectory)/terraform.tfplan'
        # run az login
        runAzLogin: true
```

Not specifying runAzLogin will default to false meaning az login will not be run
```yaml
    - task: TerraformCLI@0
      displayName: 'terraform apply'
      inputs:
        command: apply
        workingDirectory: $(test_templates_dir)
        environmentServiceName: 'env_test'
        commandOptions: '$(System.DefaultWorkingDirectory)/terraform.tfplan'
        # run az login
```

Setting to false will indicate az login should be NOT run
```yaml
    - task: TerraformCLI@0
      displayName: 'terraform apply'
      inputs:
        command: apply
        workingDirectory: $(test_templates_dir)
        environmentServiceName: 'env_test'
        commandOptions: '$(System.DefaultWorkingDirectory)/terraform.tfplan'
        # do not run az login
        runAzLogin: false
```

**Comment by scott-doyland-burrows** [comment]
Hi,

That is great - sounds like it will do the job.

Thanks.

**Comment by charleszipp** [comment]
@scott-doyland-burrows this has been published as 0.6.8 to the marketplace and should be available to your pipelines now
